_content/booklets/buddhism-in-myanmar_bischoff-roger.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Buddhism in Myanmar: A Short History"
+authors:
+  - "Roger Bischoff"
+external_url: "https://www.bps.lk/library-search-select.php?id=wh399"
+drive_links:
+  - "https://drive.google.com/file/d/1v_kALEtADIUKm19J1a2ZHZ18LbA62zuJ/view?usp=sharing"
+  - "https://drive.google.com/file/d/1jCH_RZtQb5OO42kLOD_-tKc0HchA-Ejo/view?usp=drive_link"
+formats: [pdf, epub]
+course: burmese
+tags:
+  - society
+year: 1995
+publisher: bps
+series: wheel
+number: 399
+pages: 88
```

Comment:
How'd you calculate that?  I count ~100 contentful half-pages for ~50 pages total